docs/tests/readability_alias_order.ex

```diff
@@ -0,0 +1,18 @@
+##Patterns: readability_alias_order
+##Issue: {"severity": "Info", "line": 16, "patternId": "readability_alias_order"}  
+defmodule CredoSampleModule do
+  alias App.CLI.{Bar, Baz}
```

Comment:
Most of the time you are using the multiple single line alias/require/import/use directives but here you are using the multi-alias/require/import/use syntax